Vagrantfile

```diff
@@ -1,6 +1,12 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+unless Vagrant.has_plugin?("vagrant-disksize")
```

Comment:
Did you see this `plugin auto-install` "feature"? Do we want to use it? https://unix.stackexchange.com/a/379634

> or, using auto-install logic for the plugin:
> 
> ```
> Vagrant.configure("2") do |config|
> 
>     required_plugins = %w( vagrant-vbguest vagrant-disksize )
>     _retry = false
>     required_plugins.each do |plugin|
>         unless Vagrant.has_plugin? plugin
>             system "vagrant plugin install #{plugin}"
>             _retry=true
>         end
>     end
> 
>     if (_retry)
>         exec "vagrant " + ARGV.join(' ')
>     end
> 
>     config.vm.box = "debian/stretch64" 
>     config.disksize.size = "20GB"
> end
> ```

Reply:
I'd say this will add unnecessary logic to `Vagrantfile` and installing this plugin is a one off action so IMHO: let's leave it